x-pack/solutions/observability/plugins/streams_app/public/components/stream_detail_overview/index.tsx

```diff
@@ -288,34 +290,18 @@ function QuickLinks({ definition }: { definition?: IngestStreamGetResponse }) {
 }
 
 function ChildStreamList({ definition }: { definition?: IngestStreamGetResponse }) {
-  const {
-    dependencies: {
-      start: {
-        streams: { streamsRepositoryClient },
-      },
-    },
-  } = useKibana();
   const router = useStreamsAppRouter();
 
-  const streamsListFetch = useStreamsAppFetch(
-    ({ signal }) => {
-      return streamsRepositoryClient.fetch('GET /api/streams', {
-        signal,
-      });
-    },
-    [streamsRepositoryClient]
-  );
+  const { wiredStreams } = useWiredStreams();
 
   const childrenStreams = useMemo(() => {
     if (!definition) {
       return [];
     }
-    return streamsListFetch.value?.streams.filter(
-      (d) => isWiredStreamDefinition(d) && d.name.startsWith(definition.stream.name)
-    );
-  }, [definition, streamsListFetch.value?.streams]);
+    return wiredStreams?.filter((d) => isDescendantOf(definition.stream.name, d.name));
+  }, [definition, wiredStreams]);
 
-  if (definition && childrenStreams?.length === 1) {
+  if (definition && childrenStreams?.length === 0) {
```

Comment:
caught that when updating component to use `useWiredStreams`

Reply:
Good catch, it was actually right before because `'logs'.startsWith('logs')`, so the stream itself was also a "child stream", but using the proper helpers fixed that.

Reply:
Ah right, I fixed the bug I introduced earlier then :)